blueprints/ember-cli-horizon/index.js

```diff
@@ -1,13 +1,24 @@
 /*jshint node:true*/
 module.exports = {
-    description: 'Default blueprint to be run after install.',
-
-    afterInstall: function() {
-        return this.addPackagesToProject([
-            { package: '@horizon/client' },
-            { package: 'bufferutil' },
```

Comment:
actually I shouldn't have included `ember-browserify` in my list. 😛 

Did you see my issue/question about using SystemJS instead of browserify?